src/EventTickets/ListTable/Columns/SalesAmountColumn.php

```diff
@@ -46,22 +47,26 @@ public function getCellValue($model, $locale = ''): string
     {
         $ticketTypes = [];
         foreach ($model->ticketTypes()->getAll() ?? [] as $ticketType) {
+            $salesCount = $ticketType->eventTickets()->count();
+            $ticketsAvailable = $ticketType->capacity - $salesCount;
+
             $ticketTypes[$ticketType->id] = [
                 'price' => $ticketType->price,
                 'capacity' => $ticketType->capacity,
+                'ticketsAvailable' => $ticketsAvailable,
             ];
         }
 
         $salesTotal = array_reduce(
             $model->eventTickets()->getAll() ?? [],
-                function (Money $carry, $eventTicket) use ($ticketTypes) {
-                    return $carry->add($ticketTypes[$eventTicket->ticketTypeId]['price']);
+                function (Money $carry, $eventTicket) {
+                    return $carry->add($eventTicket->amount);
             },
             new Money(0, give_get_currency())
         );
         $maxCapacitySales = array_reduce($ticketTypes, function (Money $carry, $ticketType) {
-            return $carry->add($ticketType['price']->multiply($ticketType['capacity']));
-        }, new Money(0, give_get_currency()));
+            return $carry->add($ticketType['price']->multiply($ticketType['ticketsAvailable']));
+        }, $salesTotal);
 
         $salesPercentage = $maxCapacitySales->formatToMinorAmount() > 0 ? max(
```

Comment:
This would be a good candidate for unit testing.

Not a blocker, but a `CalculateMaxCapacitySales` action with tests would increase confidence in delivery.